__tests__/nc-news4.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+const { seed } = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/topics.js");
+
+afterAll(() => {
+  return db.end;
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("The array should return 200 all ok", () => {
+    return request(app).get("/api/articles/3").expect(200);
+  });
+  test("should return 404 if article_id is not provided", () => {
```

Comment:
The path for this test is technically a correct endpoint for get all articles (next) so will not work then!